docs/src/main/sphinx/release/release-432.md

```diff
@@ -0,0 +1,57 @@
+# Release 432 (2 Nov 2023)
+
+:::{note}
+From this point forward, release notes will include a ⚠️ symbol to designate
+breaking changes. Breaking changes may:
+* Require users to update their configuration, such as if a property is removed
+* Update the default values for configuration properties, which may change
```

Comment:
Changes of default values for configuration properties that significantly change behavior of the system. Performance improvements only are not considered as breaking changes.